soc/st/stm32/stm32wb0x/Kconfig

```diff
@@ -9,8 +9,17 @@ config SOC_SERIES_STM32WB0X
 	select CPU_CORTEX_M_HAS_VTOR
 	select CPU_CORTEX_M_HAS_SYSTICK
 	select CPU_HAS_ARM_MPU
+	select HAS_PM
 	select HAS_STM32CUBE
 	select SOC_EARLY_INIT_HOOK
 	# WB0x has a ROM bootloader executed at reset,
 	# which makes the following option required
 	select INIT_ARCH_HW_AT_BOOT
+	# STM32WB0 series only supports suspend-to-RAM as low-power mode.
+	# Make sure that S2RAM support is enabled if Power Management is
+	# selected by the user. PM_DEVICE is also required because devices
+	# need to be reinitialized after resuming. CUSTOM_MARKING is enabled
+	# to use SoC-specific hardware registers for resume detection.
+	select PM_DEVICE if PM
+	select PM_S2RAM if PM
```

Comment:
This selection is redundant, `PM_S2RAM` is enabled automatically based on devicetree. With this line, it is now not possible to enable PM without enabling PM_S2RAM. This may be intentional, so not blocking, but the design allows for enabling PM but disabling the `suspend-to-ram` states in the devicetree like any other state.